app/src/main/java/pl/edu/pjatk/pamo/skrawek/ui/account/AccountFragment.java

```diff
@@ -36,4 +49,17 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         // TODO: Use the ViewModel
     }
 
+    private void fillFragmentWithData(View view) {
+        updateText(view, R.id.fullNameTextView, getProperty(NAME) + " " + getProperty(SURNAME));
+        updateText(view, R.id.cityTextView, getProperty(POSTAL_CODE) + " " + getProperty(CITY));
+        updateText(view, R.id.streetTextView, getProperty(STREET_NUMBER));
+        updateText(view, R.id.phoneTextView, getProperty(PHONE));
+        updateText(view, R.id.accountStatusTextView, getProperty(STATUS));
```

Comment:
Could you transfer the creation of an object to repositry patern?

Reply:
Within next PR if necessary.